gcputil/credentials.go

```diff
@@ -13,18 +13,19 @@ import (
 	"golang.org/x/oauth2"
 	"golang.org/x/oauth2/google"
 	"golang.org/x/oauth2/jwt"
-	googleoauth2 "google.golang.org/api/oauth2/v2"
-	"gopkg.in/square/go-jose.v2"
+	"google.golang.org/api/googleapi"
 	"io/ioutil"
 	"net/http"
+	"net/url"
 	"os"
 	"path/filepath"
-	"strings"
 )
 
 const (
-	labelRegex                 string = "^(?P<key>[a-z]([\\w-]+)?):(?P<value>[\\w-]*)$"
-	defaultHomeCredentialsFile        = ".gcp/credentials"
+	labelRegex                         string = "^(?P<key>[a-z]([\\w-]+)?):(?P<value>[\\w-]*)$"
```

Comment:
Consider `MustCompile`ing this to a package level variable. This will make any compilation issue immediately visible and take that process out of the ParseGcpLabels path.